Parse/src/main/java/com/parse/ParseLogInterceptor.java

```diff
@@ -168,44 +177,97 @@ private static String formatBytes(byte[] bytes, String contentType) {
     } else if (contentType.contains("text")) {
       return new String(bytes).trim();
     } else {
-      throw new IllegalStateException("We can not print this " + contentType);
+      return "Not supported contentType";
```

Comment:
Actually, will we ever hit this in a normal circumstance? If not, shouldn't this still be `IllegalStateException`?